test/extended/cmd.sh

```diff
@@ -74,75 +75,71 @@ popd
-docker login -u imagensbuilder -p ${IMAGE_NS_TOKEN} -e fake@example.org ${DOCKER_REGISTRY}
-oc tag --source=docker busybox:latest image-ns/busybox:latest
-oc import-image busybox
-docker pull busybox
-docker tag -f docker.io/busybox:latest ${DOCKER_REGISTRY}/image-ns/busybox:latest
-docker push ${DOCKER_REGISTRY}/image-ns/busybox:latest
-docker rmi -f ${DOCKER_REGISTRY}/image-ns/busybox:latest
-
-
-DOCKER_CONFIG_JSON=${HOME}/.docker/config.json
-oc new-project dc-ns
-oc delete all --all
-oc delete secrets --all
-oc secrets new image-ns-pull .dockerconfigjson=${DOCKER_CONFIG_JSON}
-oc secrets new-dockercfg image-ns-pull-old --docker-email=fake@example.org --docker-username=imagensbuilder --docker-server=${DOCKER_REGISTRY} --docker-password=${IMAGE_NS_TOKEN}
-
-oc process -f test/extended/fixtures/image-pull-secrets/pod-with-no-pull-secret.yaml --value=DOCKER_REGISTRY=${DOCKER_REGISTRY} | oc create -f - 
-wait_for_command "oc describe pod/no-pull-pod | grep 'Back-off pulling image'" 30*TIME_SEC
-oc delete pods --all
-
-# TODO remove sleeps once jsonpath stops panicing.  The code still works without the sleep, it just looks nasty
-
-oc process -f test/extended/fixtures/image-pull-secrets/pod-with-new-pull-secret.yaml --value=DOCKER_REGISTRY=${DOCKER_REGISTRY} | oc create -f - 
-sleep 1
-wait_for_command "oc get pods/new-pull-pod -o jsonpath='{.status.containerStatuses[0].imageID}' | grep 'docker'" 30*TIME_SEC
-oc delete pods --all
-docker rmi -f ${DOCKER_REGISTRY}/image-ns/busybox:latest
-
-
-oc process -f test/extended/fixtures/image-pull-secrets/pod-with-old-pull-secret.yaml --value=DOCKER_REGISTRY=${DOCKER_REGISTRY} | oc create -f - 
-sleep 1
-wait_for_command "oc get pods/old-pull-pod -o jsonpath={.status.containerStatuses[0].imageID} | grep 'docker'" 30*TIME_SEC
-oc delete pods --all
-docker rmi -f ${DOCKER_REGISTRY}/image-ns/busybox:latest
-
-oc process -f test/extended/fixtures/image-pull-secrets/dc-with-old-pull-secret.yaml --value=DOCKER_REGISTRY=${DOCKER_REGISTRY} | oc create -f - 
-sleep 4
-wait_for_command "oc get pods/my-dc-old-1-prehook -o jsonpath={.status.containerStatuses[0].imageID} | grep 'docker'" 30*TIME_SEC
-oc delete all --all
-docker rmi -f ${DOCKER_REGISTRY}/image-ns/busybox:latest
-
-oc process -f test/extended/fixtures/image-pull-secrets/dc-with-new-pull-secret.yaml --value=DOCKER_REGISTRY=${DOCKER_REGISTRY} | oc create -f - 
-sleep 4
-wait_for_command "oc get pods/my-dc-1-prehook -o jsonpath={.status.containerStatuses[0].imageID} | grep 'docker'" 30*TIME_SEC
-oc delete all --all
-docker rmi -f ${DOCKER_REGISTRY}/image-ns/busybox:latest
+os::cmd::expect_success 'oc new-project image-ns'
+os::cmd::expect_success 'oc delete all --all'
+os::cmd::expect_success "docker login -u imagensbuilder -p '$( oc sa get-token builder )' -e fake@example.org ${docker_registry}"
+os::cmd::expect_success 'oc tag --source=docker busybox:latest image-ns/busybox:latest'
+os::cmd::expect_success 'oc import-image busybox'
```

Comment:
@soltysh this seems to flake with the following error: 
```
FAILURE after 0.418s: test/extended/cmd.sh:109: executing 'oc import-image busybox' expecting success: the command returned the wrong error code
There was no output from the command.
Standard error from the command:
Error from server: imagestream "busybox" cannot be updated: the image stream was updated from "505" to "507"
```

Do you know what is going on here? Have you seen this before?